### DISCUSSION TITLE
文明5 无法运行

### DISCUSSION THREAD

**Discussion by github342655596** [Gamuxorg/bbs#314] [游戏或网站运行反馈]
/tmp/.mount_CiviliB3EoJ1/unionfs: error while loading shared libraries: libfuse.so.2: cannot open shared object file: No such file or directory
/tmp/.mount_CiviliB3EoJ1/AppRun: 行 23: ./Civ5XP: 没有那个文件或目录

**Comment by github342655596** [comment]
上面这个什么意思？怎么解决？

**Comment by hyh19962008** [comment]
sudo apt install libfuse2

Best

________________________________

**Comment by github342655596** [comment]
我安装了啊

**Comment by hyh19962008** [comment]
需要32位的

Best

________________________________

**Comment by github342655596** [comment]
它这次没说缺了什么，就是游戏跳不出来。我看了资源管理器，它在运行的，但是桌面上什么也没有，我也没觉得它占用了资源什么的，好像没运行一样。

**Comment by hyh19962008** [comment]
要不重启一下试试(-_-;)

Best

________________________________

**Comment by github342655596** [comment]
重启也没用，你方法我试了，我也试了很多方法，重启了好几次了，但是没用。

**Comment by hyh19962008** [comment]
解压后运行吧，./civilization.appimage --appimage-extract

Best

________________________________

**Comment by github342655596** [comment]
完全不行耶，我不管是运行Apprun还是Civ5XP都不行耶，还是重启试了一下，不行。
我看我还是下个windows的 用wine试试吧。

**Comment by hyh19962008** [comment]
可以发一下运行civ5xp的命令行输出吗？

Best

________________________________

**Comment by github342655596** [comment]
winwods版wine也不行，没用。他什么都不显示

**Comment by github342655596** [comment]
owl@linux:~/下载/squashfs-root/usr/bin$ ./Civ5
Civ5.desktop  Civ5.png      Civ5XP        
owl@linux:~/下载/squashfs-root/usr/bin$ ./Civ5XP 
owl@linux:~/下载/squashfs-root/usr/bin$

**Comment by github342655596** [comment]
什么都没有

**Comment by hyh19962008** [comment]
放弃了(_ _;)...
________________________________

**Comment by hyh19962008** [comment]
试试文明太空吧
